gnosis/conditional-markets-interface
# Feature #71: Feature Flagging for Onboarding Modes

# Description

This PR will introduce a new environment variable that has to be used from now on instead of `WHITELIST_ENABLED`

**⚠️ `WHITELIST_ENABLED` is now deprecated and won't be read anymore ⚠️**

# To Test:
- Run the interface with `ONBOARDING_MODE=DISABLED`, notice you are not longer required to be whitelisted or create a tiered whitelist account.
- Run the interface with `ONBOARDING_MODE=WHITELIST`, notice you will see the "ApplyBeta" banner, and there will be no button in the header to prompt a login
- Run the Interface with `ONBOARDING_MODE=TIERED`, notice you will not see the "ApplyBeta" banner, and there will be a button in the header to prompt a login, guiding you through the application flow.

# Code:
`travis-ci.yml` updated to include new environment variable. **On development, the `TIERED` onboarding will be rolled out for testing**
